content/en/docs/reference/glossary/service-mesh.md

```diff
@@ -6,6 +6,7 @@ managed, observable and secure communication between
 [workload instances](/docs/reference/glossary/#workload-instance).
 
 Service names combined with a namespace are unique within a mesh. For example,
+in a [multi-cluster](/docs/reference/glossary/#multi-cluster) mesh,
```

Comment:
Instead, use the {{< gloss >}} feature which gives the user a nice popup instead of a link.

Reply:
I wondered about this actually. So far it seems that we use links from glossary entries to other glossary entries, instead of glossary entries with nested glossary entries. Isn't that better. What happens in the pop-up if a glossary entry has a nested gloss entry?

Reply:
Oh, I didn't even notice that this was another glossary entry. Yeah, entry-to-entry references should use normal links as popups-in-popups won't work right.

Reply:
```suggestion
In a [multi-cluster](/docs/reference/glossary/#multi-cluster) mesh, for example,
```